doc/pages/Manual.md

```diff
@@ -994,7 +991,7 @@ files.
   dependencies of the package toward packages external to the <span class="opam">opam</span>
```

Comment:
Should this be "the package's external dependencies"?
Do the external dependencies belong to the package?